web/ui/react-app/src/pages/Rules.tsx

```diff
@@ -1,15 +1,108 @@
 import React, { FC } from 'react';
 import { RouteComponentProps } from '@reach/router';
 import PathPrefixProps from '../PathPrefixProps';
-import { Alert } from 'reactstrap';
-
-const Rules: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix }) => (
-  <>
-    <h2>Rules</h2>
-    <Alert color="warning">
-      This page is still under construction. Please try it in the <a href={`${pathPrefix}/rules`}>Classic UI</a>.
-    </Alert>
-  </>
-);
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { Alert, Table } from 'reactstrap';
+import { useFetch } from '../utils/useFetch';
+import { Link } from '@reach/router';
+
+interface Rule {
+  name: string;
+  query: string;
+  health: string;
+  evaluationTime: string;
+  lastEvaluation: string;
+  type: string;
+  lastError: string;
+}
+
+interface EnrichRule {
+  name: string;
+  file: string;
+  rules: Rule[];
+  evaluationTime: string;
+  lastEvaluation: string;
+}
+
+interface RulesMap {
+  groups: EnrichRule[];
+}
+
+const Rules: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix }) => {
+  const { response, error } = useFetch<RulesMap>(`${pathPrefix}/api/v1/rules`);
+  const roundUp = (n: number) => {
+    return Math.floor(n * 100) / 100;
+  };
+
+  if (error) {
+    return (
+      <Alert color="danger">
+        <strong>Error:</strong> Error fetching Rules: {error.message}
+      </Alert>
+    );
+  } else if (response.data) {
+    const groups: EnrichRule[] = response.data.groups;
+    return (
+      <>
+        <h2>Rules</h2>
+        {groups.map((g, i) => {
+          return (
+            <Table size="sm" striped={true} bordered={true} key={i}>
+              <thead>
+                <tr>
+                  <td colSpan={3}>
+                    <a href={'#' + g.name}>
+                      <h2>{g.name}</h2>
+                    </a>
+                  </td>
+                  <td>
+                    <h2>{roundUp(parseFloat(g.lastEvaluation))}s ago</h2>
+                  </td>
+                  <td>
+                    <h2>{g.evaluationTime}us</h2>
```

Comment:
After converting (and renaming) the evaluation duration to seconds, take a look at how the targets page formats the duration string: https://github.com/prometheus/prometheus/blob/ae93bae88fed884f59adef604ea059deff81650b/web/ui/react-app/src/pages/targets/ScrapePoolPanel.tsx#L82